charts/kubewarden-controller/templates/rbac.yaml

```diff
@@ -206,6 +206,71 @@ rules:
   - create
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: audit-scanner-cluster-role
+  labels:
+    {{- include "kubewarden-controller.labels" . | nindent 4 }}
+  annotations:
+    {{- include "kubewarden-controller.annotations" . | nindent 4 }}
+rules:
+- apiGroups:
+  - policies.kubewarden.io
+  resources:
+  - clusteradmissionpolicies
+  - admissionpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - policies.kubewarden.io
+  resources:
+  - clusteradmissionpolicies/status
+  - admissionpolicies/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+    - policies.kubewarden.io
+  resources:
+    - policyservers
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - policies.kubewarden.io
+  resources:
+    - policyservers/status
+  verbs:
+    - get
+    - list
+    - watch
```

Comment:
As we are using the same `verbs` and `apiGroups`. Cannot we merge these two entries?